LibOS/shim/src/sys/shim_mmap.c

```diff
@@ -68,8 +68,23 @@ void * shim_do_mmap (void * addr, size_t length, int prot, int flags, int fd,
         }
```

Comment:
`tmp->file` leaks here, please remove `tmp` entirely and pass `NULL` to the lookup function